ScnSoft-Education/LeetCodeTraining.Tests/PalindromeNumberTests/PalindromeNumberSolutionTests.cs

```diff
@@ -0,0 +1,93 @@
+﻿using LeetCodeTraining.PalindromeNumber;
+
+namespace LeetCodeTraining.Tests.PalindromeNumber
+{
+    public class PalindromeNumberSolutionTests
+    {
+        [Fact]
```

Comment:
Please use Theory andInlineData attributes to get rid of duplication of the test code..